pkg/cmd/openshift-service-serving-cert-signer/cmd.go

```diff
@@ -0,0 +1,108 @@
+package openshift_service_serving_cert_signer
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	"github.com/openshift/origin/pkg/cmd/server/origin"
+)
+
+type OpenShiftSSCS struct {
+	ConfigFile     string
+	KubeConfigFile string
+	Output         io.Writer
+}
+
+var longDescription = templates.LongDesc(`
+	Start the OpenShift controllers`)
+
+func NewOpenShiftServiceServingCertSignerCommand(name string, out, errout io.Writer) *cobra.Command {
```

Comment:
we are approaching the 80 character constructor names (a German complaining about word length!) ;-)